SPECS/kernel/config

```diff
@@ -510,6 +510,10 @@ CONFIG_CPU_IBPB_ENTRY=y
 CONFIG_CPU_IBRS_ENTRY=y
 CONFIG_CPU_SRSO=y
 # CONFIG_GDS_FORCE_MITIGATION is not set
+CONFIG_MITIGATION_RFDS=y
```

Comment:
On as a result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/2fb08b672eb78c4930413cf529734792456ae15f